src/phoenix/core/traces.py

```diff
@@ -153,6 +154,7 @@ def get_spans(
         start_time: Optional[datetime] = None,
         stop_time: Optional[datetime] = None,
         root_spans_only: Optional[bool] = False,
+        span_ids: Optional[Iterable[SpanID]] = None,
```

Comment:
I think it might be cleaner to have a separate method `get_spans_by_id` - otherwise this method seems a bit bloated?